content/en/blog/_posts/2019-06-13-volume-cloning-alpha.md

```diff
@@ -0,0 +1,107 @@
+
+The [Kubernetes Storage SIG](https://github.com/kubernetes/community/tree/master/sig-storage) identified clone operations as critical functionality for many stateful workloads. For example, a database administrator may want to duplicate a database volume and create another instance of an existing database.
+
+By providing a standard way to trigger clone operations in the Kubernetes API, Kubernetes users can now handle use cases like this without having to go around the Kubernetes API (and manually executing storage system specific operations).  While cloning is similair in behavior to creating a snapshot of a volume, then creating a volume from the snapshot, it's significantly more streamlined and in cases of most storage devices it is significantly more efficient for the device itself.  It also eliminates the need for another object to manage and performs the process in a single operation. 
+
+Instead, Kubernetes users are now empowered to incorporate clone operations in a cluster agnostic way into their tooling and policy with the comfort of knowing that it will work against arbitrary Kubernetes clusters regardless of the underlying storage.
+
+## Which volume plugins support Kubernetes Cloning?
+
+Kubernetes supports three types of volume plugins: in-tree, Flex, and [Container Storage Interface](https://github.com/container-storage-interface/spec/blob/master/spec.md) (CSI). See [Kubernetes Volume Plugin FAQ](https://github.com/kubernetes/community/blob/master/sig-storage/volume-plugin-faq.md) for details.
+
+Cloning is only supported for CSI drivers (not for in-tree or Flex). To use the Kubernetes cloning feature, ensure that a CSI Driver that implements cloning is deployed on your cluster.
+For a list of CSI drivers that currently support cloning see the [CSI Drivers doc](https://kubernetes-csi.github.io/docs/drivers.html).
+
+## Kubernetes API and Cloning
+
+The cloning feature in Kubernetes is enabled by a new `PersistentVolumeClaim.DataSource` field that references an existing PersistentVolumeClaim object in the same namespace.  There are no new objects introduced to enable cloning, instead the existing DataSource field in the PersistentVolumeClaim object is expanded to be able to accept the name of an existing PersistentVolumeClaim in the users namespace.  It is important to note that from a users perspective a clone is just another PersistentVolume and PersistentVolumeClaim, the only difference being that that PersistentVolume is being populated with the contents of another PersistentVolume at creation time.  After creation it behaves exactly like any other Kubernetes PersistentVolume and adheres to the same behaviors and rules.
```

Comment:
s/users perspective/user's perspective/

Reply:
s/users namespace/user's namespace/

Reply:
It might be nice to say a little something about the existing use of the `PersistentVolumeClaim.DataSource` field and how that existing use is or is not affected. I presume DataSource is used for snapshot referencing?